fighthealthinsurance/medicaid_api.py

```
@@ -0,0 +1,945 @@
+        or household_size is None
+        or monthly_income is None
+        or pregnant is None
+        or kids is None
+        or receiving_ssdi is None
+        or on_medicare is None
+    ):
+        return (eligible_2025, eligible_2026, eligible_medicare, alts, missing)
+
+    # ---- with core info present, evaluate categories ----
+    pfpl_2025 = pct_fpl(monthly_income, household_size, 2025)
+    pfpl_2026 = pct_fpl(monthly_income, household_size, 2026)
```

Ruff (0.13.1)

Local variable `pfpl_2026` is assigned to but never used

Remove assignment to unused variable `pfpl_2026`

(F841)